edu.ohio_state.khatchad.refactoring/src/edu/ohio_state/khatchad/refactoring/ConvertConstantsToEnumRefactoring.java

```diff
@@ -205,29 +204,27 @@ private Collection extractConstants(Collection col) {
 	private String getFullyQualifiedName(IJavaElement elem) {
 		// Get the associated set.
 		Collection set = null;
-		for (final Iterator it = this.computer.getEnumerizationForest()
-				.iterator(); it.hasNext();) {
+		for (final Iterator it = this.computer.getEnumerizationForest().iterator(); it.hasNext();) {
 			final Collection col = (Collection) it.next();
 			if (col.contains(elem))
 				set = col;
 		}
 
 		// Get the fully qualified type name.
-		final StringBuffer fqn = new StringBuffer(this.simpleTypeNames.get(set)
-				.toString());
+		final StringBuffer fqn = new StringBuffer(this.simpleTypeNames.get(set).toString());
 		fqn.insert(0, '.');
 		fqn.insert(0, this.packageNames.get(set));
 		return fqn.toString();
 	}
 
 	/**
-	 * @return
+	 * @return getting all the Field 
 	 */
-	IJavaProject getJavaProject() {
+	IJavaProject getJavaIField() {
```

Comment:
I'm not sure why this change is necessary. Can you explain? In fact, I don't think that this file needs to change at all.